src/elements/content-uploader/ItemAction.js

```diff
@@ -52,7 +54,7 @@ const ItemAction = ({ status, onClick, intl, isFolder = false }: Props) => {
     return (
         <div className="bcu-item-action">
             <Tooltip position="top-left" text={tooltip}>
-                <PlainButton onClick={onClick} type="button">
+                <PlainButton data-resin-target={target} onClick={onClick} type="button">
```

Comment:
Should this be used to track other states as well? If it isn't retrying should that data attribute be on the button?